### PR TITLE
Add Connect Error PHP Curl

### PR DESCRIPTION
**Pull request by mehrshaddarzi**

### CODE REVIEW

**Comment by AndyGaskell** [comment]
Hi @mehrshaddarzi 

Thanks for the PR, the try/catch here might be a good change to the code.

Did you try using the http_errors option in package options as a solution to this?
https://php-fcm.readthedocs.io/en/latest/packageoptions.html

File: src/FcmClient.php [hunk 1]
```diff
@@ -81,9 +81,13 @@ public function send(Request $request): array
         $url = $request->getUrl();
 
         // Create and send the request.
-        $response = $client->post($url, [
-            'json' => $request->getBody()
-        ]);
+        try {
+            $response = $client->post($url, [
```
Comment (Philipp91) [inline src/FcmClient.php:85]: Should we also check the HTTP Status? I have a snippet like 

```
        if ($response->getStatusCode() === 400) {
            throw new \Exception("Firebase returned 400");
        } elseif ($response->getStatusCode() === 401) {
            throw new \Exception("Firebase authentication error");
        } else {
            return $response->getStatusCode() === 200;
        }
```

**Comment by Philipp91** [comment]
> Did you try using the http_errors option in package options as a solution to this?
https://php-fcm.readthedocs.io/en/latest/packageoptions.html

Oh right. That should subsume my comment and this PR itself.

File: src/FcmClient.php (hunk 1 above)
Comment (AndyGaskell) [reply src/FcmClient.php:85]: That'd be useful, would you please add that as a subsequent PR? @Philipp91

**Comment by AndyGaskell** [comment]
The testing level drops a little because of this, I'll try add to the tests to address this

**Pull request merged**